components/GeneralNotes.js

```diff
@@ -26,18 +19,13 @@ export default function GeneralNotes({
   }
 
   return (
-    <StyledQuestionCard
-      padding={padding}
-      width={width}
-      margin={margin}
-      isLocation={isLocation}
-    >
-      <StyledQuestionLabel htmlFor="generalNotes">
+    <>
+      <StyledLabel htmlFor="generalNotes">
```

Comment:
Here, the `StyledLabel` is correct; if you change it for the file above, make sure to adjust the code for this line as well.

Reply:
To be honest I don't really understand this remark. If I have a fieldset with a legend at one place, can't I have a label at another place then? And I read that fieldsets should only be used if there are 2 or more answer options to group them together which is not the case here. So I just leave it like that.

Reply:
What I meant was that if you change the `StyledLabel` from a `label` to `fieldset`, you need to adapt this place as well. But you've done it correctly, so never mind.